### ISSUE TITLE
Stream Preview on MouseHover disabled ?

### ISSUE THREAD

**Issue by AldarisStarcraft** [night/BetterTTV#2436]
Hello,

where is this new feature that was released some days ago ? It was such a good feature, why did you choose to remove it ?

**Comment by EhsanKia** [comment]
Stream, do you mean hovering over the streams in the directory and seeing a mini player?

That works fine on my end, do you have "Directory Preview" enabled in settings?

**Comment by AldarisStarcraft** [comment]
No, I mean when I hovering one of those streams, it supposed to show a picture of it
![2017-05-10_220721](https://cloud.githubusercontent.com/assets/28606292/25918847/2959d732-35cd-11e7-9159-a484f286d7d3.png)

**Comment by EhsanKia** [comment]
A picture of the stream? I doubt that was a BTTV feature, might be something Twitch was experimenting on and you were part of the test group and they've ended the test.

**Comment by AldarisStarcraft** [comment]
Oh really ? Is that possible ? It was pretty similar with the picture preview when you hovering a link picture.
Anyway, it's a great feature, I really hope that we can all get it.
